metagraph/src/cli/load/load_graph.hpp

```diff
@@ -25,6 +26,15 @@ std::shared_ptr<Graph> load_critical_graph_from_file(const std::string &filename
 
 std::shared_ptr<graph::DeBruijnGraph> load_critical_dbg(const std::string &filename);
 
+template <class Graph>
+std::shared_ptr<graph::CanonicalDBG> primary_to_canonical(std::shared_ptr<Graph> graph,
+                                                          size_t cache_size = 1024);
+
+std::shared_ptr<const graph::DeBruijnGraph>
+make_cached_graph(const graph::DeBruijnGraph &graph,
+                  const Config &config,
+                  size_t cache_size = 1024);
```

Comment:
So do we ever need larger cache?

Reply:
If you have a batch of very redundant short reads, or a single read of length > 1024, then yes, having a bigger cache is useful. I'll increase this value